ci-operator/config/openshift/contour-operator/openshift-contour-operator-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
```

Comment:
About the file name: I don't know whether it's important or not but contour-operator uses `main` as its base branch, not `master`.

Reply:
I will change this to reflect ```main```

Reply:
One of the prompts of `make new-repo` should have asked about what default branch to use.

Reply:
And yea, we need to make sure main is used instead of master.

Reply:
Ok thanks, I will sort this out to reflect ```main```